assets/js/components/user-input/UserInputPreview.js

```diff
@@ -110,29 +109,18 @@ export default function UserInputPreview( props ) {
 
 								<UserInputPreviewGroup
 									questionNumber={ 3 }
-									title={ __(
-										'What are the goals of this site?',
-										'google-site-kit'
-									) }
-									edit={ goTo.bind( null, 3, 'user-input' ) }
-									values={ settings?.goals?.values || [] }
-									options={ USER_INPUT_ANSWERS_GOALS }
-								/>
-							</Cell>
-							<Cell lgSize={ 6 } mdSize={ 8 } smSize={ 4 }>
-								<UserInputPreviewGroup
-									questionNumber={ 4 }
 									title={ __(
 										'What do you need help most with for this site?',
```

Comment:
We need to update the copy for all three questions based on [this Figma design](https://www.figma.com/file/CdtrHB0HAjfGRWTnTMMQI0/SummaryWidget?node-id=917%3A24281).